detector/fcal/BeamCal_o1_v01_geo.cpp

```diff
@@ -45,6 +50,10 @@ static DD4hep::Geometry::Ref_t create_detector(DD4hep::Geometry::LCDD& lcdd,
   const double bcalInnerR = dimensions.inner_r();
   const double bcalOuterR = dimensions.outer_r();
   const double bcalInnerZ = dimensions.inner_z();
+  if (bcalInnerZ/dd4hep::mm > 3200) {
```

Comment:
I put this warning in on request by Sergej Schuwalow. The reason is that space should be provided after BeamCal for equipment such as vacuum valves, kickers, etc., which I am not sure is implemented in lcgeo. There is an agreement with the MDI group that BeamCal should be positioned not further than 3200 mm and this is why we need this warning. But if you are sure that all equipment is implemented in lcgeo, I can of course remove the warning.